### ISSUE TITLE
About "Sata_remap" parameter setting

### ISSUE THREAD

**Issue by cocoakeer** [fbelavenuto/arpl#345]
Hello, developer.
In which menu does the parameter configuration "Sata_remap" take effect? I found this parameter invalid after configuration.
您好，开发者。
请问"Sata_remap"这个参数配置在那个菜单中生效？我配置后发现此参数无效。

**Comment by fbelavenuto** [comment]
In this case uses the "sata_remap" variable:
sata_remap: (source port)>(dest port)
For exampre, 0>9 put the first disk (Disk 0 in DSM) to 10th (disk 9 in DSM)
Other example: 0>9:9>0 change places of disks

**Comment by cocoakeer** [comment]
> In this case uses the "sata_remap" variable: sata_remap: (source port)>(dest port) For exampre, 0>9 put the first disk (Disk 0 in DSM) to 10th (disk 9 in DSM) Other example: 0>9:9>0 change places of disks

OK, I will try again. Thank you very much.

**Comment by fbelavenuto** [comment]
More information here:
https://gugucomputing.wordpress.com/2018/11/11/experiment-on-sata_args-in-grub-cfg/

**Comment by cocoakeer** [comment]
> More information here: https://gugucomputing.wordpress.com/2018/11/11/experiment-on-sata_args-in-grub-cfg/

I have a question. Which menu do I add this parameter to? Cmdline or syninfo?
<img width="364" alt="image" src="https://user-images.githubusercontent.com/112269603/203542457-9b52d69e-fe67-495b-891d-3483f881bfaa.png">

**Comment by fbelavenuto** [comment]
Cmdline.
I'm working on documentation: https://github.com/fbelavenuto/arpl/wiki/Add-an-cmdline-synoinfo-entry

**Comment by fbelavenuto** [comment]
Ok, please post the full screenshot of "Show SATA(s) # ports and drives"

**Comment by cocoakeer** [comment]
> > Ok, please post the full screenshot of "Show SATA(s) # ports and drives"
> 
> <img alt="image" width="765" src="https://user-images.githubusercontent.com/112269603/203553490-724dae3c-d027-4e8e-ba57-c32a93d4a194.png">
> 
> The first controller is the boot disk. I have successfully hidden it The second controller is the second hard disk, 00 The third controller recognizes six sata ports, and the first two must be placed at the end. 第一个控制器为引导盘，我已经成功隐藏 第二个控制器为第二块硬盘，00 第三个控制器识别6个sata口，前两个需要放到最后。

This motherboard is a beggar board. There are only 4 sata interfaces in physics, but there are 6 logical interfaces, so I need to adjust the order of the hard disk
此主板为乞丐板，物理只有4个sata接口，但逻辑识别为6个，所以我要调整硬盘顺序。

**Comment by fbelavenuto** [comment]
Uses only the following values:
SataPortMap: 146
DiskIdxMap: 0A0004

(Removes sata_remap)

**Comment by cocoakeer** [comment]
> 

My second controller only has one hard disk. Can I set it this way?
我第二个控制器只有一块硬盘，可以这么设置吗？

**Comment by fbelavenuto** [comment]
> > 
> 
> My second controller only has one hard disk. Can I set it this way? 我第二个控制器只有一块硬盘，可以这么设置吗？

Yes, but you wish put SataPortMap with 116 and DiskIdxMap with 0A0001

**Comment by cocoakeer** [comment]
> > > 
> > 
> > 
> > My second controller only has one hard disk. Can I set it this way? 我第二个控制器只有一块硬盘，可以这么设置吗？
> 
> Yes, but you wish put SataPortMap with 116 and DiskIdxMap with 0A0001

I want the second controller to be 00 and the third controller to be 01, so that the DS918 can be displayed in order normally.
我希望第二个控制器为00，第三个控制器为01，这样在DS918中就可以正常的按照顺序显示。

**Comment by cocoakeer** [comment]
> > > 
> > 
> > 
> > My second controller only has one hard disk. Can I set it this way? 我第二个控制器只有一块硬盘，可以这么设置吗？
> 
> Yes, but you wish put SataPortMap with 116 and DiskIdxMap with 0A0001

如果我按照SataPortMap=116；DiskIdxMap=0A0001配置，我发现我识别不到后面两个硬盘，我曾经这么尝试过。
If I follow SataPortMap=116; For DiskIdxMap=0A0001 configuration, I found that I could not identify the last two hard disks, which I had tried.
